chart/kubeapps/Chart.yaml

```diff
@@ -31,4 +31,4 @@ maintainers:
 name: kubeapps
 sources:
   - https://github.com/vmware-tanzu/kubeapps
-version: 10.3.3-dev0
+version: 10.3.3
```

Comment:
The currently released version is still 10.3.1, it seems there is an ongoing issue on their side:

````
helm repo update && helm search repo kubeapps
NAME                    CHART VERSION   APP VERSION     DESCRIPTION
bitnami/kubeapps        10.3.1          2.5.0           Kubeapps is a web-based UI for launching and ma...
````